.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -9,7 +9,19 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
looking at the missing signature process -- does it make sense to create a macOS binary, if it can't be run?